.github/workflows/spelling.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/download-artifact@v2
+    - uses: dawidd6/action-download-artifact@v2
```

Comment:
For my own curiosity: what does this action have that makes us change to it?

Reply:
The previous one doesn't support downloading artifacts from a different workflow.